projects/ngx-formentry/src/form-entry/form-factory/form-control.service.ts

```diff
@@ -142,7 +142,8 @@ export class FormControlService {
     this.wireCalculator(
       question,
       control,
-      form ? form.dataSourcesContainer.dataSources : null
+      form,
+      form.dataSourcesContainer.dataSources
```

Comment:
Suggestion: Use optional chaining to safely access these nested objects and their properties.